docs/ECLLanguageReference/ECLR_mods/ExtrSvcs-ExternalServicesImpl.xml

```diff
@@ -162,6 +168,28 @@
             </row>
 
             <row>
+              <entry><emphasis>FOLD<indexterm>
+                  <primary>FOLD</primary>
```

Comment:
Yes, probably. I'll make sure there is a JIRA open for documenting the signing and access rights changes.